packages/contract-wrappers/src/utils/market_orders_optimization_utils.ts

```diff
@@ -0,0 +1,44 @@
+import { SignedOrder } from '@0xproject/types';
+import * as _ from 'lodash';
+
+import { constants } from './constants';
+
+export const marketOrdersOptimizationUtils = {
```

Comment:
I think this should be renamed to something like `calldataOptimizationUtils`, since there are functions that aren't market buys or sells that we will also eventually optimize (like `matchOrders`).